### ISSUE TITLE
GDAL compilation failed in CI workflow

### ISSUE THREAD

**Issue by manuGil** [CUSP-Urban-Science-and-Policy/Citizen-Voice#39]

**Comment by manuGil** [comment]
Compiling GDAL fails when pulling library form ubuntu-gis. The C compiler failed with the message: 

```shell
g++ -pthread -shared -Wl,--rpath=/opt/hostedtoolcache/Python/3.10.6/x64/lib -Wl,--rpath=/opt/hostedtoolcache/Python/3.10.6/x64/lib build/temp.linux-x86_64-cpython-310/extensions/gnm_wrap.o -L../../.libs -L../../ -L/opt/hostedtoolcache/Python/3.10.6/x64/lib -L/usr/lib -lgdal -o build/lib.linux-x86_64-cpython-310/osgeo/_gnm.cpython-310-x86_64-linux-gnu.so
      error: command '/usr/bin/gcc' failed with exit code 1
```

https://github.com/CUSP-Urban-Science-and-Policy/Citizen-Voice/actions/runs/3053968143/jobs/4925269870

**Comment by manuGil** [comment]
Fixed by #41
